config.json

```diff
@@ -20,6 +20,17 @@
       ]
     },
     {
+      "uuid": "dc4ad67d-0fe4-0880-5a13-3aebb8759bcc3095de1",
+      "slug": "two-fer",
+      "core": false,
```

Comment:
I was actually thinking that this would be a nice "bonus" exercise that you get as soon as you submit Hello World.

By not including it as core and not having it be unlocked by anything, it's immediately available to work on in addition to Leap.